File: docs/_docs/code-snippets/java/src/main/java/org/apache/ignite/snippets/BaselineTopology.java
package org.apache.ignite.snippets;

import org.apache.ignite.Ignite;
import org.apache.ignite.Ignition;
import org.apache.ignite.cluster.ClusterState;

public class BaselineTopology {

    void activate() {
        //tag::activate[]
        Ignite ignite = Ignition.start();

        ignite.cluster().state(ClusterState.ACTIVE);; 
        //end::activate[]
    }
    
    void enableAutoadjustment() {
        //tag::enable-autoadjustment[]
        
        Ignite ignite = Ignition.start();

        ignite.cluster().baselineAutoAdjustEnabled(true);
                
        ignite.cluster().baselineAutoAdjustTimeout(30000);

        //end::enable-autoadjustment[]

        //tag::disable-autoadjustment[]
        ignite.cluster().baselineAutoAdjustEnabled(false);
        //end::disable-autoadjustment[]

    }
}

File: docs/_docs/code-snippets/dotnet/BaselineTopology.cs
using System;
using Apache.Ignite.Core;

namespace dotnet_helloworld
{
    public static class BaselineTopology
    {
        public static void Activate()
        {
            // tag::activate[]
            IIgnite ignite = Ignition.Start();
            ignite.GetCluster().SetActive(true);
            // end::activate[]
        }

        public static void EnableAutoAdjust()
        {
            // tag::enable-autoadjustment[]
            IIgnite ignite = Ignition.Start();
            ignite.GetCluster().SetBaselineAutoAdjustEnabledFlag(true);
            ignite.GetCluster().SetBaselineAutoAdjustTimeout(30000);
            // end::enable-autoadjustment[]
        }

        public static void DisableAutoAdjust()
        {
            IIgnite ignite = Ignition.Start();
            // tag::disable-autoadjustment[]
            ignite.GetCluster().SetBaselineAutoAdjustEnabledFlag(false);
            // end::disable-autoadjustment[]
        }
    }
}
